CoJezdzi/AppFramework/_Networking/Request/UMWarsawTransportApi.swift

```diff
@@ -0,0 +1,32 @@
+import APIKit
+
+final class UMWarsawTransportApi {
+    
+    private init() {
+    }
+    
+    struct GetTransportMeanRequest: UMWarsawRequest {
+        
+        let vehicleType: WarsawVehicleType
+
+        // MARK: RequestType
+        typealias Response = UMWarsawTransportResponse
+        
+        var method: HTTPMethod {
+            return .post
+        }
+        
+        var path: String {
+            return "/action/busestrams_get"
+        }
+        
+        var parameters: Any? {
```

Comment:
Why not `[URLQueryItem]?` :)

Reply:
To make things ASAP I've only based on APIKit.Request protocol properties. I changed it to 
  `  var queryParameters: [String: Any]? { get }`

Reply:
better done than perfect is my middle name ;)